charts/latest/blob-csi-driver/values.yaml

```diff
@@ -20,6 +20,9 @@ image:
     tag: v1.1.0
     pullPolicy: IfNotPresent
 
+imagePullSecrets: []
```

Comment:
it's better write like this:
https://github.com/helm/charts/blob/master/stable/cert-manager/values.yaml#L5-L9

Reply:
i don't know what i'm looking at? It looks the same to me?
Are you referring to the comments?

Reply:
yes, just comments, it's a little better

Reply:
Done